### DISCUSSION TITLE
Is there a way to unset set_root?

### DISCUSSION THREAD

**Discussion by WantenMN** [nvim-tree/nvim-tree.lua#2663] [Q&A] [answered]
In nvim-tree, pressing `.` sets the selected folder as the root, making folders above this "root" invisible. Is there a way to unset set_root? Something like no_set_root?

My current approach is to use `:q` to exit nvim and then re-enter.

**Comment by alex-courtis** [comment, marked as answer]
`.` is mapped by default to `run_command`, which simply opens command line mode with the file.

Do you have `.` mapped to something else?

Regardless, you can unmap in your `on_attach`; see `:help nvim-tree-mappings`

**Reply by WantenMN** [reply]
I feel very sorry, I mixed up `nvim-tree.lua` and `neo-tree.nvim`, and asked the question in the wrong place. I apologize for wasting your time.

Thank you for your response and assistance. I will try using `nvim-tree.lua`.

I apologize again!
